packages/@ama-sdk/core/project.json

```diff
@@ -27,7 +27,7 @@
         "script": "build:esm2015"
       },
       "dependsOn": [
-        "compile"
```

Comment:
Why did you change the default `compile` name which comes with default config?

Reply:
There is no technical reason behind the change. I changed it to be aligned on naming with the other configs.